integration/assumptions/main_test.go

```diff
@@ -0,0 +1,13 @@
+package assumptions_test
+
+import (
+	"testing"
+
+	"github.com/databricks/cli/integration/internal"
+)
+
+// TestMain is the entrypoint executed by the test runner.
+// See [internal.Main] for prerequisites for running integration tests.
+func TestMain(m *testing.M) {
+	internal.Main(m)
```

Comment:
@pietern we might want to re-think this - it turns out having TestMain disables go test's built-in caching, which could be useful in speeding up test runs both locally and on CI.